README.md

```diff
@@ -1,3 +0,0 @@
-# Nectar
```

Comment:
Please pull from upstream after PR #1 is merged so this file isn't deleted.

Reply:
Rebased